demos/smart_framing_demo/cpp_gapi/main.cpp

```diff
@@ -0,0 +1,344 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <chrono>
+#include <string>
+#include <vector>
+
+#include <gflags/gflags.h>
+#include <opencv2/gapi/streaming/cap.hpp>
+#include <opencv2/gapi/imgproc.hpp>
+#include <opencv2/gapi/core.hpp>
+
+#include <opencv2/gapi/fluid/core.hpp>
+#include <opencv2/gapi/fluid/imgproc.hpp>
+#include <opencv2/gapi/cpu/core.hpp>
+#include <opencv2/gapi/cpu/imgproc.hpp>
+#include <opencv2/highgui.hpp>
+
+#include <openvino/openvino.hpp>
+
+#include <monitors/presenter.h>
+#include <utils_gapi/stream_source.hpp>
+#include <utils/args_helper.hpp>
+#include <utils/config_factory.h>
+#include <utils/images_capture.h>
+#include <utils/ocv_common.hpp>
+#include <utils/performance_metrics.hpp>
+#include <utils/default_flags.hpp>
+#include <models/detection_model.h>
+
+#include "custom_kernels.hpp"
+
+
+namespace util {
+
+static cv::gapi::GKernelPackage getKernelPackage(const std::string& type) {
+    if (type == "opencv") {
+        return cv::gapi::combine(cv::gapi::core::cpu::kernels(),
+                                 cv::gapi::imgproc::cpu::kernels());
+    } else if (type == "fluid") {
+        return cv::gapi::combine(cv::gapi::core::fluid::kernels(),
+                                 cv::gapi::imgproc::fluid::kernels());
+    } else {
+        throw std::logic_error("Unsupported kernel package type: " + type);
+    }
+    GAPI_Assert(false && "Unreachable code!");
+}
+
+} // namespace util
+
+namespace {
+constexpr char h_msg[] = "show the help message and exit";
+DEFINE_bool(h, false, h_msg);
+
+constexpr char i_msg[] =
+"an input to process. The input must be a single image, a folder of images, video file or camera id. Default is 0";
+DEFINE_string(i, "0", i_msg);
+
+constexpr char camera_resolution_msg[] = "set camera resolution in format WxH.";
+DEFINE_string(res, "1280x720", camera_resolution_msg);
+
+constexpr char yolo_m_msg[] = "path to an .xml file with a trained YOLO v4 Tiny model.";
+DEFINE_string(m_yolo, "", yolo_m_msg);
```

Comment:
```suggestion
constexpr char m_msg[] = "path to an .xml file with a trained YOLO v4 Tiny model.";
DEFINE_string(m, "", m_msg);
```
Could you make it just `m`: `--m_yolo`->`-m` because it's the primal model of the demo